RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -582,3 +582,133 @@ void auto_autumnatonQuest()
 
 	return;
 }
+
+boolean auto_haveTrainSet()
+{
+	return auto_get_campground() contains $item[model train set] && auto_is_valid($item[model train set]); //check if the model train set is in the campground
+}
+
+void auto_modifyTrainSet(int one, int two, int three, int four, int five, int six, int seven, int eight)
+{
+	string page = `choice.php?pwd&whichchoice=1485&option=1&slot[0]={one}&slot[1]={two}&slot[2]={three}&slot[3]={four}&slot[4]={five}&slot[5]={six}&slot[6]={seven}&slot[7]={eight}`;
+	visit_url(page,true,true);
+	visit_url("main.php");
+	return;
+}
+
+void auto_checkTrainSet()
+{
+	if(!auto_haveTrainSet()) return;
+	/* A list of what the station numbers are (thanks Zdrvst for compiling this list for your CS script)
+	1: meat
+	2: mp regen
+	3: all stats
+	4: hot res, cold dmg
+	5: stench res, spooky dmg
+	6: wood, joiners, or stats (orc chasm bridge stuff)
+	7: candy
+	8: double next stop
+	9: cold res, stench dmg
+	11: spooky res, sleaze dmg
+	12: sleaze res, hot dmg
+	13: monster level
+	14: mox stats
+	15: basic booze
+	16: mys stats
+	17: mus stats
+	18: food drop buff
+	19: copy last food drop
+	20: ore
+	*/
+	int one = 8; //doubler
+	int two;
+	int three;
+	if((my_level() < 13 || get_property("auto_disregardInstantKarma").to_boolean())) //check if we need more stats
```

Comment:
Thoughts on doubling smut orc parts instead of stats since that is what the swap criteria is based on? Train gives so many stats, I bet we would get quite high in level even not doubled

Reply:
I get to level 15+ most runs and that is including swapping to cold medicine cabinet after bridge parts is done with the Workshed PR. I think a good solution would probably be to double stats until we got to level 12, then swap to doubling bridge parts. I know, not necessarily consistent with the rest of Autoscend going until 13, but like you said, there are so many stats. I'll think about how to implement that

Reply:
I think this is a decent solution